File: src/main/java/com/dongfg/project/api/graphql/type/Comment.java
package com.dongfg.project.api.graphql.type;

import lombok.Data;
import org.springframework.data.annotation.Id;

import java.util.Date;

/**
 * @author dongfg
 * @date 2017/10/15
 */
@Data
public class Comment {
    @Id
    private String id;
    private String comment;
    private String name;
    private String email;
    private String clientIp;
    private Date createTime;
}

File: src/main/kotlin/com/dongfg/project/api/web/graphql/type/Comment.kt
package com.dongfg.project.api.web.graphql.type

import org.springframework.data.mongodb.core.mapping.Document
import java.time.LocalDateTime
import javax.persistence.Id

/**
 * @author dongfg
 * @date 2018/3/19
 */
@Document
data class Comment(
        @field: Id var id: String?,
        var comment: String,
        var name: String,
        var email: String?,
        var clientIp: String?,
        var createTime: LocalDateTime? = LocalDateTime.now()
)